core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Hm, I thought the learner dashboard was fully usable on mobile. @apb7 @Arunabh98 is this not the case?

If there is a specific part of it that's not usable on mobile then let's rename this file to reflect that appropriately (e.g. learnerDashboardFeedbackThreads or something like that).

Reply:
OK, seems fine to me as long as the non-mobile version consists exclusively of the non-mobile parts and the name reflects that.

Reply:
Great, I will rename this suit then.